### PR TITLE
Camera fixes, and send data more often.

### PR DESCRIPTION
**Pull request by KTibow**
- Send Fully Motion every 500ms instead of 5000ms
- Make camera base64 be in jpeg to be smaller
- Make camera use proper aspect ratio

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -358,7 +358,7 @@ class BrowserMod {
     this._motionTimeout = setTimeout(() => {
       this._fullyMotion = false;
       this.update();
-    }, 5000);
+    }, 500);
```
Comment (thomasloven) [inline js/main.js]: This is not how often updates are sent. It's the cooldown. When motion is detected, that's sent immediately. But if no motion is detected, the clear signal is not sent for five seconds. Most motion detectors have something like this, but usually with a much longer time.

Comment (KTibow) [inline js/main.js]:
```suggestion
    }, 5000);
```

Comment (KTibow) [reply js/main.js]: Fixed. Sorry for not understanding.

**Comment by thomasloven** [comment]
I'll trust you know what you're doing, because I don't.

**Pull request merged**